src/Uno.UI/UI/Xaml/Media/Imaging/RenderTargetBitmap.skia.cs

```diff
@@ -43,46 +43,58 @@ private static (int ByteCount, int Width, int Height) RenderAsBgra8_Premul(UIEle
 			var renderSize = element.RenderSize;
 			var visual = element.Visual;
 
-			if (renderSize is { IsEmpty: true } or { Width: 0, Height: 0 })
+			var previousClip = visual.LayoutClip;
+
+			try
 			{
-				return (0, 0, 0);
-			}
+				// Remove any existing layout clip, we want to render the full element, not
+				// the clipped part based on the existing parent's layout slot.
+				visual.LayoutClip = null;
```

Comment:
I Initially didn't like hot-fixing the clip here and I thought it would be a lot nicer to send this as a parameter to RenderRootVisual itself. But actually, this is exactly what we have to do because only the "layout clip" should be removed but not the clip from UIElement.Clip (I've tested on WinUI), so if we send a parameter to RenderRootVisual to ignore the clipping altogether for the "root element", then that's actually incorrect.